### ISSUE TITLE
Display WordPress Version on Simple Sites

### ISSUE THREAD

**Issue by cuemarie** [Automattic/wp-calypso#75978]
### What

Display the WordPress Version on simple sites under Settings -> General, like we do for AT sites.

### Why

TLDR - This is very foundational information that a site owner might want to know, and for some reason, it's much harder to find on a "simple" site than it is on AT.  It seems like we're forcing simple site owners to rely on support to provide them with this information (either via chat/email/forums, depending on their plan, or through our docs), when we could automatically show them this and remove that dependency.

For example, if a simple/free site owner wanted to know what version of WordPress their site uses, here are the likely paths that user can follow:

Path 1

1. Search their site dashboard and find nothing
2. Contact Support or the Forums - creating an unnecessary contact when this information could probably be easily shown.

Path 2

1. Search their site and find nothing
2. Search our Support Documents (or contact support)
3. Support Docs would likely lead them here: [Check Your WordPress Version](https://wordpress.com/support/check-your-wordpress-version/)
4. This guide doesn't actually say the version, and instead directs non-plugin-enabled-site-owners to another guide: [PHP Environment](https://wordpress.com/support/php-environment/)

Also, at time of reporting, this guide says WordPress 6.1.1 - it's possible this is actually out of date?

<img width="957" alt="Screen Shot 2023-04-19 at 12 15 50 PM" src="https://user-images.githubusercontent.com/27249804/233177522-c9b39691-7b39-4378-984a-d9058c029fcf.png">

### How

Display the WordPress Version for simple sites in the Settings > General page, just like AT sites (see below), and update when new versions of WordPress are rolled out to those sites:

![Markup on 2023-04-19 at 12:13:06](https://user-images.githubusercontent.com/27249804/233177709-bb52dac4-26a2-4da4-be34-fed90f6d28e2.png)

**Comment by cuemarie** [comment]
Asked for insight/discussion here: p1681932370939649-slack-C02T4NVL4JJ

**Comment by cuemarie** [comment]
PR https://github.com/Automattic/wp-calypso/pull/75983 in the works ⬆️  🤞

**Comment by cuemarie** [comment]
Looks like this is likely a no, in terms of user-facing info. See pMz3w-hrH-p2

I've also asked here: p1683153618564629/1681932370.939649-slack-C02T4NVL4JJ if something internal-facing only might be possible - we'll see!
